src/pages/[slug].tsx

```diff
@@ -0,0 +1,49 @@
+import type { GetStaticPaths, GetStaticProps } from 'next'
+import client from '@/lib/contentful'
+import { type TypeLandingPageSkeleton } from '@/contentful/types'
+import Page from '@/components/common/Page'
+import { type LandingPageEntry } from '@/config/types'
+
+const LandingPage = (props: { landingPage: LandingPageEntry }) => {
+  if (!props.landingPage) return null
+
+  return <Page {...props} />
+}
+
+export const getStaticProps: GetStaticProps = async ({ params }) => {
+  const slug = params?.slug as string
```

Comment:
`slug` can be `undefined` and this is not being handled. It would be better to do something like:

```suggestion
  const slugParam = params?.slug
  
  if (!slugParam) {
    return {
      notFound: true
    }
  }
  
  const slug = Array.isArray(slugParam) ? slugParam[0] : slugParam
```

I can see that this is done elsewhere in the project so I will leave it up to you whether to change this.

Reply:
I remember we've discussed this in some other PR. I feel here the check is unnecessary. Reason being this code runs at build time and if there is no `slug` the page is not built and the error will be visible.